patches/0007-Add-backend-code-gen.patch

```diff
@@ -392,7 +264,12 @@ index 00000000000000..1cc4952e7ca9c2
 +		"The "+expTag.String()+" tag is specified, but other tags required to enable that backend were not met.",
 +		"Required build tags:",
 +		"  "+backend.constraint.String(),
-+		"Please check your build environment and build command for a reason one or more of these tags weren't specified.")
++		"Please check your build environment and build command for a reason one or more of these tags weren't specified.",
++		"",
++		"If you only performed a Go toolset upgrade and didn't expect this error, your code was likely depending on fallback to Go standard library crypto.",
++		"As of Go 1.21, Go crypto fallback is a build error. This helps prevent accidental fallback.",
++		"Removing "+backend.name+"crypto will restore pre-1.21 behavior by intentionally using Go standard library crypto.",
```

Comment:
Note that I'm not mentioning `allowcryptofallback` here. I think it's probably exceptionally rare that someone actually *needs* to use it rather than just removing the crypto backend they aren't using. I don't want to inadvertently encourage people to try it out without understanding the context--I think it would be better to describe it in the FIPS readme where there's more room for detail and we can update it more freely.